qa/tasks/radosgw_admin_rest.py

```diff
@@ -812,4 +812,35 @@ def task(ctx, config):
 
     # TESTCASE 'ratelimit' 'global' 'modify' 'anonymous' 'enabled' 'succeeds'
     (ret, out) = rgwadmin_rest(admin_conn, ['ratelimit', 'modify'], {'ratelimit-scope' : 'bucket', 'global': 'true', 'enabled' : 'true'})
-    assert ret == 200
\ No newline at end of file
+    assert ret == 200
+
+    # TESTCASE 'bucket-chown','bucket','modify','administrative user','succeeds'
+
+    # create a first bucket with foo user
+    foo_bucket = connection.create_bucket('foo_bucket')
+
+    # create 201 objects in foo_bucket
+    for i in range(201):
+        key = boto.s3.key.Key(foo_bucket)
```

Comment:
this just creates an in-memory key to represent an object name. see https://boto.cloudhackers.com/en/latest/ref/s3.html#boto.s3.key.Key

you'll need to name these objects, like `key = boto.s3.key.Key(foo_bucket, 'obj%s' % i)`, then call something like `key.set_contents_from_string('')` to actually upload then